docs/gettsim_developer/how-to-maintain.rst

```diff
@@ -103,21 +110,31 @@ Procedure
 ^^^^^^^^^
 
 In the following we will consider an example where GETTSIM's stable version is
-``0.2.0``. Version ``0.3.0`` is currently developed on the master branch. There is a
-maintenance branch ``0.2`` to receive patches for the ``0.2.x`` line of releases. And a
-critical bug was found, which should be fixed in both ``0.3.0`` and in ``0.2.1``.
+``0.2.0``. Version ``0.3.0`` is currently under development on the main branch. There is
+a maintenance branch ``0.2`` to receive patches for the ``0.2.x`` line of releases. And
+a critical bug was found, which should be fixed in both ``0.3.0`` and in ``0.2.1``.
 
-1. Create a PR containing the bug fix which targets the master branch.
+1. Create a PR containing the bug fix which targets the main branch.
 2. Add a note to the release notes for version 0.2.1.
-3. Squash merge the PR into master and note down the commit sha.
+3. Squash merge the PR into main and note down the commit sha.
 4. Create a new PR against branch ``0.2``. Call the branch for the PR
    ``backport-pr[No.]-to-0.2.1`` where ``[No.]`` is the PR number.
 5. Use ``git cherrypick -x <commit-sha>`` with the aforementioned commit sha to apply
    the fix to the branch. Solve any merge conflicts, etc..
 6. Add the PR to the milestone for version ``0.2.1`` so that all changes for a new
    release can be collected.
-7. The release process for a patch version works as above in :ref:`releases` to release
-   ``0.2.1``; just that it is released off the maintenance branch, not off master.
+7. The release process for a patch version works as above in :ref:`releases`, steps
```

Comment:
Is there a reason why this is under "How to Release Patched Versions". I would have expected it directly under "How To Release Major or Minor Versions"

Reply:
What do you mean by "this"? Everything *does* appear above, too -- I just tried to make the process for patches more self-contained relative to what it was. Both of us ended up confused at some point during 0.4.1 / 0.4.2 :smile:

Reply:
I mean point 7. (The release process for a patch version)

Patch versions are usually just "normal" releases and not necessarily backports to previously released versions (which to my understanding is the main topic of this latter section).

Reply:
> Patch versions are usually just "normal" releases and not necessarily backports to previously released versions (which to my understanding is the main topic of this latter section).

There is the misunderstanding, I think. We should (continue to) think about them as patches = backports, not as adding new features. Thinking a bit ahead here when we reach more of a stationary state, I agree that, e.g., 0.4.1 was different.

Anyhow, we might switch to something like the year-indication anyhow, so leaving it as is for now and no need for big discussions ATM.